pyscf/adc/radc.py

```diff
@@ -0,0 +1,1751 @@
+# Copyright 2014-2019 The PySCF Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Samragni Banerjee <samragnibanerjee4@gmail.com>
+#         Alexander Sokolov <alexander.y.sokolov@gmail.com>
+#
+
+'''
+Restricted algebraic diagrammatic construction
+'''
+import time
+import numpy as np
+from pyscf import lib
+from pyscf.lib import logger
+from pyscf.adc import radc_ao2mo
+from pyscf import __config__
+
+def kernel(adc, nroots=1, guess=None, eris=None, verbose=None):
```

Comment:
If the RADC kernel is the same as the UADC, we should use the RADC kernel in UADC